e2e/README.md

```diff
@@ -0,0 +1,13 @@
+# E2E Test for this sample app
```

Comment:
actually I guess if you include the deploy step it's not bad, I'll give that a shot